fly/commands/watch.go

```diff
@@ -11,8 +11,9 @@ import (
 )
 
 type WatchCommand struct {
-	Job   flaghelpers.JobFlag `short:"j" long:"job"   value-name:"PIPELINE/JOB"   description:"Watches builds of the given job"`
-	Build string              `short:"b" long:"build"                               description:"Watches a specific build"`
+	Job       flaghelpers.JobFlag `short:"j" long:"job"   	value-name:"PIPELINE/JOB"   description:"Watches builds of the given job"`
+	Build     string              `short:"b" long:"build"                               	description:"Watches a specific build"`
+	Timestamp bool                `short:"u" long:"timestamps"								description:"Print with timestamp"`
```

Comment:
This flag's a bit sneaky but that tends to happen with short flags. Fine with me since `-u` seems unlikely to collide with much at least.

Reply:
Just noticed that the `build` command has `-t` as a short form for team 😶

Reply:
Heh, sure let's just go with `-t` then.